### PR TITLE
Title changed Changes done

### PR DESCRIPTION
**Pull request by speechara**
Changes done

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ function Dashboard() {
       <Sidebar />
       <div className="dashboard-content">
         <div className="dashboard-box">
-          <h3>Number of users till now</h3>
```
Comment (skothagattu0304) [inline src/components/Dashboard/Dashboard.js:129]: change it to number of users till date

**Review by skothagattu0304 — COMMENTED** [review]
make changes and resent pull request

File: src/components/Dashboard/Dashboard.js (hunk 1 above)
Comment (speechara) [reply src/components/Dashboard/Dashboard.js:129]: ok

File: src/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ function Dashboard() {
           <p>{numUsers}</p>
         </div>
         <div className="dashboard-box">
-          <h3>Number of orders till now</h3>
+          <h3>Number of orders till today</h3>
```
Comment (skothagattu0304) [inline src/components/Dashboard/Dashboard.js:133]: make it until

**Review by skothagattu0304 — APPROVED** [review]

**Pull request merged**